cpp/include/cudf/types.hpp

```diff
@@ -169,7 +169,7 @@ enum type_id {
   INT64,      ///< 8 byte signed integer
   FLOAT32,    ///< 4 byte floating point
   FLOAT64,    ///< 8 byte floating point
-  BOOL8,      ///< Boolean using one byte per value, 0 == false, else true
```

Comment:
Why remove documentation if the type ID is staying?

Reply:
This was an oversight. I deleted it at one point and when I added back clearly forgot the comment. Fixed.